src/PolyhedralGeometry/PolyhedralFan/standard_constructions.jl

```diff
@@ -303,3 +303,39 @@ function Base.:*(PF1::PolyhedralFan{QQFieldElem}, PF2::PolyhedralFan{QQFieldElem
   prod = Polymake.fan.product(pm_object(PF1), pm_object(PF2))
   return PolyhedralFan{QQFieldElem}(prod, QQ)
 end
+
+#################################################################################
+## Hyperplane arrangements
+#################################################################################
+
+@doc raw"""
```

Comment:
Docstrings should start with the function signature, indented like this (so it is formatted as code) :-)
```suggestion
@doc raw"""
    matrix_to_arrangement(A::MatElem{<: FieldElem})

```